docs/stack_config.rst

```diff
@@ -39,6 +39,44 @@ A list of arbitrary shell or python commands or scripts to run. Find out more in
 
 A dictionary of key-value pairs to be supplied to a CloudFormation or Troposphere template as parameters. The keys must match up with the name of the parameter, and the value must be of the type as defined in the template. Note that Boto3 throws an exception if parameters are supplied to a template that are not required by that template. Resolvers can be used to add functionality to this key. Find out more in the `Resolvers`_ section.
 
+A parameter can be specified either as a single value/resolver or a list of values/resolvers. Lists of values/resolvers will be formatted into an AWS compatible comma separated string e.g. "value1,value2,value3". Lists can contain a mixture of values and resolvers.
+
+Syntax:
+
+.. code-block:: yaml
+
+  parameters:
+    <parameter1_name>: "value"
+    <parameter2_name>: !<resolver_name> <resolver_value>
+    <parameter3_name>:
+      - "value1"
+      - "value2"
+      - "value3"
+    <parameter4_name>:
+      - !<resolver_name> <resolver_value>
+      - !<resolver_name> <resolver_value>
+      - !<resolver_name> <resolver_value>
+    <parameter5_name>:
+      - !<resolver_name> <resolver_value>
+      - "value1"
+      - !<resolver_name> <resolver_value>
+
+Example:
+
+.. code-block:: yaml
+
+    parameters:
+      database_username: "mydbuser"
+      database_password: !environment_variable DATABASE_PASSWORD
+      subnet_ids:
+        - "subnet-12345678"
+        - "subnet-87654321"
+        - "subnet-11111111"
+      security_group_ids:
+        - "sg-12345678"
+        - !stack_output security-groups::BaseSecurityGroupId
+        - !file_contents /file/with/security_group_id.txt
```

Comment:
I think there's value in having all three here - good to show that different resolvers can be used together.